test/suite/test_schema01.py

```diff
@@ -78,11 +78,6 @@ def cursor(self, config=None):
         return self.session.open_cursor(self.tablename, None, config)
 
     def test_populate(self):
-        # We skip testing the tiered storage scenarios as we fail to create
-        # column groups in tiered storage scenarios. We should fix this issue
-        # and then remove the condition to skip tests. FIXME: WT-9048
```

Comment:
WT-9058 is now fixed. I've confirmed this and `test_schema02` are now passing when we remove the `skipTest` logic

Reply:
Did you mean WT-9048?

Reply:
Yep, comment fixed 👍